packages/clerk-js/src/utils/captcha.ts

```diff
@@ -0,0 +1,64 @@
+import { loadScript } from './script';
+
+interface RenderOptions {
+  sitekey: string;
+  callback?: (token: string) => void;
+  'error-callback'?: (err: any) => void;
+}
+
+declare global {
+  export interface Window {
+    turnstile: { execute: (container?: string | HTMLElement | null, params?: RenderOptions) => void };
+  }
+}
+
+const SCRIPT_URL = 'https://challenges.cloudflare.com/turnstile/v0/api.js?render=explicit';
+
+export async function loadCaptcha() {
+  return new Promise(resolve => {
+    resolve(
+      loadScript(SCRIPT_URL, {
+        defer: true,
+        globalObject: window.turnstile,
+      }),
+    );
+  }).then(() => {
+    const turnstile = window.turnstile;
+
+    console.log('captcha loaded');
+    return turnstile;
+  });
+}
+
+export const getCaptchaToken = async () => {
+  let captchaToken = '' as string | unknown;
```

Comment:
❓ Why are we casting here ?

Reply:
@panteliselef removed this and handled properly the returned type of `handleCaptchaTokenGeneration`